concepticondata/conceptlists/KBA-2021-912.tsv

```diff
@@ -0,0 +1,913 @@
+KBA-2021-912-707	707	m541-k-bp-fl-fa	seventeen	2453	SEVENTEEN		dix-sept	siebzehn	sewentien	lesome le bosupa		7. Concepts of order  7A. Quantities and Measure	M498		
+KBA-2021-912-708	708	m542-k-bp-fl-fa	eighteen	1713	EIGHTEEN		dix-huit	achtzehn	agtien	lesome le borobabobedi		7. Concepts of order  7A. Quantities and Measure	M499		
+KBA-2021-912-709	709	m543-k-bp-fl-fa	nineteen	1714	NINETEEN		dix-neuf	neunzehn	negentien	lesome le borobabongwe		7. Concepts of order  7A. Quantities and Measure	M500		
+KBA-2021-912-710	710	m544-k-bp-fl-fa	twenty	1710	TWENTY		vingt	zwanzig	twintig	masome a mabedi		7. Concepts of order  7A. Quantities and Measure	M501		
+KBA-2021-912-711	711	m546-k-bp-fl-fa	thirty	1715	THIRTY		trente	dreißig	dertig	masome a mararo		7. Concepts of order  7A. Quantities and Measure	M503		
+KBA-2021-912-712	712	m547-k-bp-fl-fa	fourty	1716	FORTY		quarante	vierzig	veertig	masome a mane		7. Concepts of order  7A. Quantities and Measure	M504		
+KBA-2021-912-713	713	m548-k-bp-fl-fa	fifty	1717	FIFTY		cinquante	fünfzig	vyftig	masome a matlhano		7. Concepts of order  7A. Quantities and Measure	M505		
+KBA-2021-912-714	714	m549-k-bp-fl-fa	sixty	1718	SIXTY		soixante	sechzig	sestig	masome a borataro		7. Concepts of order  7A. Quantities and Measure	M506		
+KBA-2021-912-715	715	m55-k-bp-fl-fa	to get, receive	694	GET		obtenir, recevoir	bekommen, erhalten	kry, bekom	go tsaya		1C. Social and Antisocial Behaviour	M151		
+KBA-2021-912-716	716	m550-k-bp-fl-fa	seventy	1721	SEVENTY		soixante-dix	siebzig	sewentig	masome a supa		7. Concepts of order  7A. Quantities and Measure	M507		
+KBA-2021-912-717	717	m551-k-bp-fl-fa	eighty	1722	EIGHTY		quatre-vingt(s)	achtzig	tagtig	masome a borobabobedi		7. Concepts of order  7A. Quantities and Measure	M508		
+KBA-2021-912-718	718	m552-k-bp-fl-fa	ninety	1724	NINETY		quatre-vingt-dix	neunzig	negentig	masome a borobabongwe		7. Concepts of order  7A. Quantities and Measure	M509		
+KBA-2021-912-719	719	m553-k-bp-fl-fa	one-hundred	1634	HUNDRED		cent	hundert, einhundert	honderd	lekgolo		7. Concepts of order  7A. Quantities and Measure	M510		
+KBA-2021-912-720	720	m554-k-bp-fl-fa	to measure	1506	MEASURE		mesurer	messen	meet	go tlhola bokete		7. Concepts of order  7A. Quantities and Measure	M511		
+KBA-2021-912-721	721	m555-k-bp-fl-fa	weight	2466	WEIGHT		poids	Gewicht	gewig	bokete		7. Concepts of order  7A. Quantities and Measure	M512		
+KBA-2021-912-722	722	m556-k-bp-fl-fa	line (row)	1048	LINE		ligne	Linie, Reihe	lyn, linie, ry	tsela		7. Concepts of order  7A. Quantities and Measure	M514		
+KBA-2021-912-723	723	m557-k-bp-fl-fa	boundary (frontier)	1010	BOUNDARY		frontière, limite	Grenze	grens			7. Concepts of order  7A. Quantities and Measure	M513		
+KBA-2021-912-724	724	m558-k-bp-fl-fa	end				bout, extrêmité, fin	Ende	end, einde	bofelelo		7. Concepts of order  7A. Quantities and Measure	M521		
```

Comment:
Maybe 742 END (OF SPACE)? I mean, in a way, counting could be conceptualized in space, going from the smallest to the biggest values (or the other way round).